packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -222,17 +225,21 @@ const node = new appmesh.VirtualNode(this, 'node', {
   accessLog: appmesh.AccessLog.fromFilePath('/dev/stdout'),
 });
 
-const virtualService = new appmesh.VirtualService(this, 'service-1', {
+const virtualService1 = new appmesh.VirtualService(this, 'service-1', {
   virtualServiceProvider: appmesh.VirtualServiceProvider.virtualRouter(router),
-  virtualServiceName: 'service1.domain.local',
+  virtualServiceName: virtualServiceName1,
+});
+const virtualService2 = new appmesh.VirtualService(this, 'service-2', {
+  virtualServiceProvider: appmesh.VirtualServiceProvider.virtualRouter(router),
+  virtualServiceName: virtualServiceName2,
```

Comment:
Can you actually create a separate example, explaining what problem does the new method in `Backend` solve? It probably deserves its own section, honestly. Let's leave this example unchanged.